Architect-Coders/MU8
# Issue #52: Data Binding on Comics Activity

**Data Binding on Comics Activity** #52 

- Delete comicsUiModel because implemented databinding.
- Create diferent bindingAdapter for use in xml.
- Create Event for use navigation.
- Create extension function for toast.

**Testing Done:**

- Huawei p20 Lite

**How to test:**

- Run the application and click onComic Category
- You can see the loading while the comics have been loaded
- Then you can see the comics in a scrollable list

**Signed-off-by**:

- Santiago Lozada <santiagolozada95@gmail.com>
